InternetShop/src/main/java/dao/UserDaoImpl.java

```diff
@@ -6,18 +6,37 @@
 import java.util.Map;
 
 public class UserDaoImpl implements UserDao {
-    private Map<String, User> usersList = new HashMap<>();
+
+    private Map<String, User> users = new HashMap<>();
 
     @Override
     public void setUserIntoDatabase(User user) {
+        users.put(user.getName(), user);
+        System.out.println("User " + user.getName() + " added to the database");
+    }
+
+    @Override
+    public User getUserByName(String name) {
+        return users.get(name);
     }
 
     @Override
-    public User getUserFromDatabase() {
-        return null;
+    public Map<String, User> getAll() {
```

Comment:
Passing map outside means anyone can change it, like, user from menu. I would either return copy of map or delete this method at all